core/network/helpers/protobuf_message_read_writer.hpp

```diff
@@ -43,8 +43,14 @@ namespace kagome::network {
      */
     template <typename MsgType>
     void read(ReadCallback<MsgType> &&cb) const {
+      if constexpr (std::is_same_v<MsgType, network::BlocksResponse>) {
+        std::ignore = [] { return 1; }();
```

Comment:
Oh, didn't know about it.

Reply:
It is from std::tuple implementation, but useful for eating any result to avoid warnings.

Reply:
Any way this peace of code was used for debugging and will be romoved.